### PR TITLE
Create clients to work like the HealthVault OneSDK.

### PR DESCRIPTION
**Pull request by mikenevi**

### CODE REVIEW

File: healthvault/src/androidTest/java/com/microsoft/healthvault/HVSettings.java [hunk 1]
```diff
@@ -46,7 +46,7 @@ public void setShellUrl(String url) {
 	@Override
 	public String getRestUrl() {
 		// TODO Auto-generated method stub
-		return null;
+		return "https://data.ppe.microsofthealth.net";
```
Comment (anbutle) [inline healthvault/src/androidTest/java/com/microsoft/healthvault/HVSettings.java]: Some kind of config or at least constants file to reference things like urls will be cleaner

File: healthvault/src/main/java/com/microsoft/healthvault/HealthServiceInstance.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+ *  DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package com.microsoft.healthvault;
+
+import java.net.URL;
+
+public class HealthServiceInstance {
+    private String id;
```
Comment (anbutle) [inline healthvault/src/main/java/com/microsoft/healthvault/HealthServiceInstance.java]: Other files are hungarian notation, these are not, should unify.

**Review by anbutle — APPROVED** [review]

File: healthvault/src/androidTest/java/com/microsoft/healthvault/HVSettings.java (hunk 1 above)
Comment (mikenevi) [reply healthvault/src/androidTest/java/com/microsoft/healthvault/HVSettings.java]: reverting this one. We mostly use the values from TestSettings.java anyway.

File: healthvault/src/main/java/com/microsoft/healthvault/HealthServiceInstance.java (hunk 2 above)
Comment (mikenevi) [reply healthvault/src/main/java/com/microsoft/healthvault/HealthServiceInstance.java]: fixed.

**Pull request merged**